apps/opik-backend/src/main/java/com/comet/opik/infrastructure/aws/IamAuthTokenRequestV2.java

```diff
@@ -0,0 +1,68 @@
+package com.comet.opik.infrastructure.aws;
+
+import software.amazon.awssdk.auth.credentials.AwsCredentials;
+import software.amazon.awssdk.auth.signer.Aws4Signer;
+import software.amazon.awssdk.auth.signer.params.Aws4PresignerParams;
+import software.amazon.awssdk.http.SdkHttpFullRequest;
+import software.amazon.awssdk.http.SdkHttpMethod;
+import software.amazon.awssdk.regions.Region;
+
+import java.net.URI;
+import java.time.Duration;
+import java.time.Instant;
+
+public class IamAuthTokenRequestV2 {
```

Comment:
@thiagohora why not just use native `IAMAuthTokenRequest` object

Reply:
This is from AWS SDK v1, we use AWS SDK V2